static/src/stylesheets/module/signup/_signup-page.scss

```diff
@@ -54,35 +54,58 @@
             color: #ffffff;
             padding: 0 .3rem;
         }
+    }
+
+    .email-sub__lozenge {
+        border-color: $features-support-3;
+        border-radius: 100px;
+        float: left;
+        width: 70%;
+        height: 3em;
+        margin-bottom: $gs-baseline;
+
+        &:hover,
+        &:focus,
+        &:active {
+            border-color: darken($features-support-3, 10%);
+        }
 
-        .email-sub__lozenge {
-            border-color: $features-support-3;
-            border-radius: 100px;
-            float: left;
-            width: 70%;
-            height: 3em;
-            margin-bottom: $gs-baseline;
-
-            &:hover,
-            &:focus,
-            &:active {
-                border-color: darken($features-support-3, 10%);
-            }
-
-            @include mq($until: tablet) {
-                width: 100%;
-            }
+        @include mq($until: tablet) {
+            width: 100%;
         }
+    }
 
-        .email-sub__submit-input {
-            width: 25%;
-            margin: 0px 0px $gs-baseline 3%;
-            background-color: inherit;
+    .email-sub__label {
+        color: #ffffff;
+        width: 100%;
+        line-height: 3;
+        pointer-events: none;
+    }
 
-            @include mq($until: tablet) {
-                width: 100%;
-                margin-left: 0;
-            }
+    .email-signup__svg--envelope__svg {
+        fill: #ffffff;
+        width: 28px;
```

Comment:
I would probably use a value from the grid system unless that doesn't work visually. Then I would usually leave a note explaining that this value is important and why.

Em values are really useful for icons. Especially if you have text that changes size between breakpoints. You only have to set one em value for the icons and they will always look in proportion. If this is the case I would probably go back to using the `em` value, but if not then make a note :)